backend/src/test/java/io/debezium/configserver/ValidateSqlSeverDatabasesIT.java

```diff
@@ -5,48 +5,39 @@
  */
 package io.debezium.configserver;
 
-import com.fasterxml.jackson.databind.node.ObjectNode;
 import io.debezium.configserver.rest.ConnectorURIs;
 import io.debezium.configserver.util.Infrastructure;
 import io.debezium.configserver.util.SqlServerInfrastructureTestProfile;
-import io.debezium.testing.testcontainers.ConnectorConfigurationTestingHelper;
+import io.debezium.testing.testcontainers.Connector;
+import io.debezium.testing.testcontainers.ConnectorConfiguration;
 import io.quarkus.test.junit.QuarkusTest;
 import io.quarkus.test.junit.TestProfile;
 import io.restassured.http.ContentType;
 
 import org.junit.jupiter.api.Test;
 
-import java.util.Map;
-
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.hasItems;
 
 @QuarkusTest
 @TestProfile(SqlServerInfrastructureTestProfile.class)
 public class ValidateSqlSeverDatabasesIT {
 
     @Test
     public void testMultipleDatabases() {
-        ObjectNode config = ConnectorConfigurationTestingHelper.getConfig(
-                Infrastructure.getSqlServerConnectorConfiguration(1)
-                        .with("database.hostname", "localhost")
-                        .with("database.port", Infrastructure.getSqlServerContainer().getMappedPort(1433))
-                        .with("database.names", "testdb,testdb2")
-        );
+        ConnectorConfiguration config = Infrastructure.getSqlServerConnectorConfiguration(1)
+                .with("database.names", "master,model");
```

Comment:
@Naros @rk3rn3r I was enabling this test for 2.0 to support multiple databases and I am getting this error. 
```
{
    "error_code": 400,
    "message": "Connector configuration is invalid and contains the following 1 error(s):\nOnly a single database name is currently supported\nYou can also find the above list of errors at the endpoint `/connector-plugins/{connectorType}/config/validate`"
}
```

Reply:
That's definitely a strange error and not what I see.  When I run the test suite I actually get
```
{
    "error_code": 400,
    "message": "Connector configuration is invalid and contains the following 1 error(s):\nUnable to connect. Check this and other connection properties. Error: The database 'master' is not enabled for Change Data Capture. Ensure that the correct database context is set and retry the operation. To report on the databases enabled for Change Data Capture, query the is_cdc_enabled column in the sys.databases catalog view.\nYou can also find the above list of errors at the endpoint `/connector-plugins/{connectorType}/config/validate`"
}
```
This makes sense because you cannot enabled CDC on `master` and this check was in fact added as a part of DBZ-4346.  I believe you need to readjust the databases to be `testdb,testdb2` since that is what we initialize the database with.

Reply:
That's definitely a strange error and not what I see.  When I run the test suite I actually get
```
{
    "error_code": 400,
    "message": "Connector configuration is invalid and contains the following 1 error(s):\nUnable to connect. Check this and other connection properties. Error: The database 'master' is not enabled for Change Data Capture. Ensure that the correct database context is set and retry the operation. To report on the databases enabled for Change Data Capture, query the is_cdc_enabled column in the sys.databases catalog view.\nYou can also find the above list of errors at the endpoint `/connector-plugins/{connectorType}/config/validate`"
}
```
This makes sense because you cannot enabled CDC on `master` and this check was in fact added as a part of DBZ-4346.  I believe you need to readjust the databases to be `testdb,testdb2` since that is what we initialize the database with.

Reply:
@Naros Did the tests pass locally for you? This time the error 
```
{
    "error_code": 400,
    "message": "Connector configuration is invalid and contains the following 1 error(s):\nUnable to connect. Check this and other connection properties. Error: The database 'testDB2' is not enabled for Change Data Capture. Ensure that the correct database context is set and retry the operation. To report on the databases enabled for Change Data Capture, query the is_cdc_enabled column in the sys.databases catalog view.\nYou can also find the above list of errors at the endpoint `/connector-plugins/{connectorType}/config/validate`"
}
```